openpype/hosts/traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -0,0 +1,767 @@
+import os
+import re
+import csv
+import clique
+from copy import deepcopy, copy
+
+from openpype.client import get_asset_by_name
+from openpype.pipeline.create import get_subset_name
+from openpype.pipeline import CreatedInstance
+from openpype.lib import FileDef, BoolDef
+from openpype.lib.local_settings import get_openpype_username
+from openpype.lib.transcoding import (
+    VIDEO_EXTENSIONS, IMAGE_EXTENSIONS
+)
+
+from openpype.hosts.traypublisher.api.plugin import (
+    TrayPublishCreator
+)
+
+
+class IngestCSV(TrayPublishCreator):
```

Comment:
I do miss the editorial information.
```suggestion
class IngestEditorialCSV(TrayPublishCreator):
```

Reply:
Well, this is the problem since this is in reality not ingesting editorial data, it doesn't make sense to have Editorial in the name.